libafl/src/executors/timeout.rs

```diff
@@ -136,23 +113,47 @@ impl<E> TimeoutExecutor<E> {
     /// This should usually be used for `InProcess` fuzzing.
     pub fn new(executor: E, exec_tmout: Duration) -> Self {
         let milli_sec = exec_tmout.as_millis();
-        let it_value = Timeval {
```

Comment:
We can probably remove `Timeval` if we don't use it anymore

Reply:
woops